crates/ere-risc0/src/compile.rs

```diff
@@ -0,0 +1,111 @@
+use crate::error::CompileError;
+use build_utils::docker;
+use risc0_zkvm::Digest;
+use serde::{Deserialize, Serialize};
+use std::{
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+use tempfile::TempDir;
+use tracing::info;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Risc0Program {
+    // TODO: Seems like the risc0 compilation is also compiling
+    // TODO: the analogous prover and verifying key
```

Comment:
Can you make an issue for this?

Reply:
Ah actually I didn't know what this mean, it's just moved from the original compile logic.

Could you elaboate more about this? Because the output of `cargo risczero build` seems to be only the elf, I didn't see the prover and verifying key.

Reply:
I see, as far as I see I the build command just calls `cargo build ...` internally, doens't do other preprocess thing, so I think it'd be fine to remove these comments (will do so in futuer PR)